doc/liblouis.texi

```diff
@@ -663,7 +663,11 @@ capsletter 6
 @opcode{begcapsword, dots}
 The dot pattern which begins a block of capital letters at the
 beginning or within a word. The block is automatically terminated
-by any character that is not a capital letter, e.g. small letters, punctuation, numbers etc.
+by any character that is not a capital letter, e.g. small letters,
+punctuation, numbers etc.
+
+In grade 2 tables, you can prevent the contraction of capital blocks
+by using the @opcoderef{capsnocont} opcode.
```

Comment:
Oops. I was too quick with my comment. Actually I meant the `capsmodechars` opcode, which is relevant if you talk about which characters terminate the capital block. `capsnocont` is not really relevant here. Sorry!